cmake/AlembicArnold.cmake

```diff
@@ -74,11 +74,6 @@ ENDIF()
 FIND_PATH( ALEMBIC_ARNOLD_INCLUDE_PATH ai.h
            PATHS
            "${ALEMBIC_ARNOLD_ROOT}/include"
-           NO_DEFAULT_PATH
-           NO_CMAKE_ENVIRONMENT_PATH
-           NO_CMAKE_PATH
-           NO_SYSTEM_ENVIRONMENT_PATH
-           NO_CMAKE_SYSTEM_PATH
            DOC "The directory where ri.h resides" )
```

Comment:
It's seems 'ri.h' was not changed to 'ai.h' in DOC string after copy-pasting from RenderMan finder.